packages/wonder-blocks-data/src/hooks/__tests__/use-hydratable-effect.test.js

```diff
@@ -407,7 +413,7 @@ describe("#useHydratableEffect", () => {
             rerender({requestId: "ID2"});
 
             // Assert
-            expect(result.current).toStrictEqual(Status.loading());
+            expect(result.current).toStrictEqual(Status.success("GOODDATA"));
```

Comment:
Why did this behavior change?

Reply:
Because the previous test case was actually copypasta and not doing what the test title stated. I updated it to have cached data for the new request ID to assert that it does indeed do what the test title says.